app/assets/stylesheets/language.scss

```diff
@@ -101,3 +103,14 @@
     background-color: #f5f5f5;
   }
 }
+
+@media (max-width: 400px) {
```

Comment:
Yes, I tried this and others combinations, but dont works.
That's why I added the class ```.col-languages```

Below you can see the same example:
http://jsfiddle.net/DTcHh/15071/

Bootstrap is keeping 2 columns (```.col-xs-6```) even on small screens